### PR TITLE
Issues with echo

### PR DESCRIPTION
**Pull request by gabbumms**
Content of shortcode function is always appearing at the top of the page. Researched and found that echo causes this but when I use return it outputs nothing. I've tried using "." and assigning the return parts to different variables to output. Is there another way to go about this? Also wordpress doesn't seem to like lines 82 and 85 with the return in particular.

### CODE REVIEW

**Comment by gabbumms** [comment]
Not sure why changes aren't showing on dev. It says it's applying the class and I upload the stye sheets to fetch but nothing really changes.

File: src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ function output_community_list($atts){
                   $old_location = $location;
               }
 
-              $output .= "<li type='disc'><a href=\"". get_the_permalink(). "\">". get_the_title(). "</a></li>";
+              $output .= "<li type=disc><a href=\"". get_the_permalink(). "\">". get_the_title(). "</a></li>";
```
Comment (bholaday) [inline src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php]: Remove `type=disc` and put any style changes in the stylesheet.

File: src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ function output_community_list($atts){
   $old_location = "";
   if ( $the_query->have_posts() ) {
       $output = "";
-      $output .= '<ul style= "list-style-type:none; padding-left:0">';
+      $output .= '<ul class="community-list__ul">';
```
Comment (bholaday) [inline src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php]: remove `__ul` from the class name

File: src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php (hunk 1 above)
Comment (gabbumms) [reply src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php]: Put this in a new class for just li in new push

File: src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php (hunk 2 above)
Comment (gabbumms) [reply src/themes/purdue-alumni-association/function-includes/shortcode-community-index.php]: done

**Comment by bholaday** [comment]
@gabbumms I made a few small changes on this. Make sure you pull those down before working on this again.

**Pull request merged**